### DISCUSSION TITLE
Üç basamaklı rakamları birbirinden farklı rakamlar programı

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus.github.io#46] [Soru Cevap]
Üç basamaklı rakamları birbirinden farklı tüm sayıları ekranda gösteren ve bu kurala uygun kaç tane sayı olduğunu söyleyen programı yazınız. Örnek çıktı : 102 103 104 105 106 107 108 109 120 123 . . . . . 980 981 982 983 984 985 986 987

**Comment by TahaGuzell** [comment]
```
for i in range(100,990):
    i=i+1
    birler=i%10
    onlar=(i//10)%10
    yüzler=i//100
    if birler!=onlar and birler !=yüzler and onlar!=yüzler:
        print(i)
```
